package.json

```diff
@@ -55,10 +55,10 @@
     "pathe": "^1.0.0"
   },
   "devDependencies": {
+    "@nuxt/eslint-config": "^0.1.1",
     "@nuxt/module-builder": "0.2.1",
     "@nuxt/schema": "3.3.2",
     "@nuxt/test-utils": "3.3.2",
-    "@nuxtjs/eslint-config-typescript": "12.0.0",
```

Comment:
Is https://github.com/nuxt/eslint-config/tree/main/packages/eslint-config#typescript outdated / can it be removed from the README? I did not notice the difference in `nuxt/` vs `nuxtjs/` :see_no_evil: